README.md

```diff
@@ -63,7 +63,7 @@ from jax import grad, jit, vmap
 def predict(params, inputs):
   for W, b in params:
     outputs = jnp.dot(inputs, W) + b
-    inputs = jnp.tanh(outputs)
+    outputs = jnp.tanh(outputs)
```

Comment:
This code is correct as written (the outputs of one iteration become the inputs of the next iteration)

Reply:
Ah! I think my brain wasn't registering the for loop for some reason. Thanks for clarifying.

Reply:
This has been reverted in the latest commit.